schematic/schemas/generator.py

```diff
@@ -613,8 +616,8 @@ def get_json_schema_requirements(self,
                 break
 
 
-        print("========================================================================================")
-        print("JSON schema successfully generated from schema.org schema!")
+        logger.info("========================================================================================")
```

Comment:
I personally don't think these lines are helpful. Maybe play with indentation to indicate structure in the logs.

Reply:
wait you don't like my ornamentation style?! :)

Reply:
yes, feel free to remove